test/intl402/NumberFormat/prototype/resolvedOptions/roundingMode.js

```diff
@@ -9,7 +10,7 @@ features: [Intl.NumberFormat-v3]
 var options;
 
 options = new Intl.NumberFormat([], {}).resolvedOptions();
-assert.sameValue(options.roundingMode, 'halfExpand (default)');
+assert.sameValue(options.roundingMode, 'halfExpand');
```

Comment:
This should use the third argument of `sameValue`, e.g.

```js
assert.sameValue(options.roundingMode, 'halfExpand', 'default');
```

Doing so will disambiguate failures from a subsequent assertion.

Reply:
Sounds good! Changed.